docker/Dockerfile.gnupg

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y \
+    python3 \
```

Comment:
Because I based this on @srfraser's Dockerfile which used ubuntu:16.04 :)

... Sure, looks easy enough trying to use a python3 base instead.